src/editor/index.php

```diff
@@ -118,7 +132,8 @@ function simplifyPath( $path ) {
 }
 
 // .. as a fully qualified filename
-$strSchemaFQFilename = realpath( simplifyPath( $strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename) );
+// $strSchemaFQFilename = realpath( simplifyPath( $strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename) );
+$strSchemaFQFilename = realpath($strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename );
```

Comment:
Can you provide a simple example that reproduces the problem (I admit that I didn´t understand the problem yet: Is simplifyPath something like realpath, but does not allow to "breakout" the config folder?), because simplifyPath isn´t working in openHAB2 environment. And so i have to find another solution than just removing it.